kibana/tests/kibana_test.py

```diff
@@ -355,4 +355,12 @@ def test_override_imagePullPolicy():
 
     r = helm_template(config)
     c = r['deployment'][name]['spec']['template']['spec']['containers'][0]
-    assert c['imagePullPolicy'] == 'Always'
\ No newline at end of file
+    assert c['imagePullPolicy'] == 'Always'
+
+  def test_adding_pod_labels():
```

Comment:
This indentation here is causing the test to fail:

```
17:57:06 ==================================== ERRORS ====================================
17:57:06 ____________________ ERROR collecting tests/kibana_test.py _____________________
17:57:06 /usr/local/lib/python2.7/site-packages/_pytest/python.py:445: in _importtestmodule
17:57:06     mod = self.fspath.pyimport(ensuresyspath=importmode)
17:57:06 /usr/local/lib/python2.7/site-packages/py/_path/local.py:701: in pyimport
17:57:06     __import__(modname)
17:57:06 E     File "/app/kibana/tests/kibana_test.py", line 360
17:57:06 E       def test_adding_pod_labels():
17:57:06 E                                   ^
17:57:06 E   IndentationError: unindent does not match any outer indentation level
17:57:06 !!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
17:57:06 =========================== 1 error in 0.24 seconds ============================
```

```suggestion
def test_adding_pod_labels():
```